src/ripple/app/misc/NetworkOPs.h

```diff
@@ -164,6 +164,10 @@ class NetworkOPs
     // Account functions
     //
 
```

Comment:
This is a key class - and it's full of undocumented methods too.  Can we get comments for this new one at least?

In particular, what's `startAfter`, `hint`, `limit`?  

It's a shame we're continuing this tradition of having callbacks for something that's just a loop.  We should in the future be creating iterators for this purpose...  not your fault of course.